WordPress/Classes/ViewRelated/Gutenberg/GutenbergMediaInserterHelper.swift

```diff
@@ -154,9 +180,6 @@ class GutenbergMediaInserterHelper: NSObject {
     }
 
     func syncUploads() {
-        if mediaObserverReceipt != nil {
```

Comment:
This duplicates the code from https://github.com/wordpress-mobile/WordPress-iOS/pull/21352 because these changes need to be tested together.